specs/init.md

```diff
@@ -33,13 +33,31 @@ it's own service, make sure it's always running by re-spawning (if needed), a `o
 - All services logs are written directly to kernel ring-buffer
   - Optionally later on, one of the daemon can be responsible of reading the logs and push them somewhere else.
 - once a service update it's status, other `waiting` threads (that depends on this one) will get freed to take start.
-## Controlling:
+
+## Controlling
```

Comment:
I think this section still miss some command to restart a service. 
In the case of an upgrade, after the new binary has been downloaded, we need to be able to restart the service.

Reply:
Yes, we can add that of course